### PR TITLE
Allow Packagist installs

### PR DESCRIPTION
**Pull request by szepeviktor**
Please do not kill Composer usage.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Directories
 /.circleci export-ignore
-/.git export-ignore
```
Comment (szepeviktor) [inline .gitattributes:3]: `.git` directory is never exported.

Comment (szepeviktor) [reply .gitattributes:3]: Try issuing this command.
`git archive HEAD | tar --list`

**Comment by jspellman814** [comment]
Thanks again for your contribution. Composer usage will certainly not be killed. We do need this line for our automation to prevent `.git` from being shipped to [.org](https://wordpress.org/plugins/wp-redis/) via our [action](https://github.com/pantheon-systems/wp-redis/actions). We do not, however, ship this configuration to .org, so anything you might be using this for you'd have to manage yourself.

**Comment by szepeviktor** [comment]
@jspellman814 There is another change
`/composer.json export-ignore`
that prevents using this repository with Composer.

**Comment by jspellman814** [comment]
We will push an update for this.

**Comment by jspellman814** [comment]
@szepeviktor This repository can still be installed using Composer.

Screen Shot 2022-11-30 at 2 35 56 PM

**Comment by szepeviktor** [comment]
@jspellman814 I wouldn't like to push my agenda.
`composer.json` has the information about this package.
You are at the wheel!

**Comment by szepeviktor** [comment]
e.g. its type

**Pull request closed without merging**